### PR TITLE
chore: Improved quick-start.md

### PR DESCRIPTION
**Pull request by Abubakrce19**
This changes the minor but crucial grammatical and spelling errors.

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs
- **Why was this change needed?** (You can also link to an open issue here)

- **Other information**:

### CODE REVIEW

**Comment by Abubakrce19** [comment]
@p-fernandez 
please have a look
Thank you

File: docs/docs/overview/quick-start.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ await novu.trigger('<REPLACE_WITH_EVENT_NAME_FROM_ADMIN_PANEL>', {
 
 The `subscriberId` is a custom identifier used when identifying your users within the Novu platform. We suggest using your internal DB identifier for this field.
 
-Novu will create an upsert command, and either create a subscriber with specified payload, or update the existing subscriber with the passed information.
+Novu will create an upsert command, and either create a subscriber with a specified payload, or update the existing subscriber with the passed information.
```
Comment (p-fernandez) [inline docs/docs/overview/quick-start.md:109]:
```suggestion
Novu will create an upsert command, and either create a subscriber with the specified payload, or update the existing subscriber with the passed information.
```

**Review by p-fernandez — CHANGES_REQUESTED** [review]

**Comment by Abubakrce19** [comment]
@p-fernandez 
Changes done

**Review by florian-lefebvre — APPROVED** [review]

**Review by ainouzgali — APPROVED** [review]
Thanks @Abubakrce19 !

**Comment by Abubakrce19** [comment]
> Thanks @Abubakrce19 !

Thank you soo much for approving on weekend too.
I have to merged PR to novu and this one (approved) but still this is showing that i am nor eligible for the swags.
Pleasen help me resolve this matter.
image

**Comment by Abubakrce19** [comment]
@ainouzgali 
Please merge my Pr before october ends.
I am not counted for swags yet.

**Pull request merged**